algorithms/CPlusPlus/Arrays/boyer_more.cpp

```diff
@@ -0,0 +1,66 @@
+//majorty element of an array 
+// element with frequency > (n/2) {n: size of array}
+//time complexity : O(n) || space complexity : O(1)
+
+
+#include<iostream>
+#include<stdio.h>
+
+
+using namespace std;
+
+
+void ismaj(int arr[],int n,int exp_maj){
+
+    int i=0;
+    int len = n;
+
+    int count =0;
+    for(i=0;i<len;i++){
+        if(arr[i]==arr[exp_maj]){
+            count++;
+        }
+    }
+
+    if(count>(n/2)){
+        cout<<"majority element : "<<arr[exp_maj];
+    }
+    else{
+        cout<<"no majority element found\n";
+    }
+}
+
+
+int majelement(int arr[],int n){ // maj element function
```

Comment:
add the function description i.e.
> // maj element function
 // function to find  major element in an array

Reply:
updated